javascript/src/main/utils/fetch.js

```diff
@@ -11,6 +11,8 @@ export async function fetchWithToken(url, getToken, options) {
   });
   if (response.status >= 400 && response.status < 600) {
     throw new Error(response.error_description);
+    // const json = await response.json();
+    // throw new Error("HTTP Error " + json.status + ": " + json.message);
```

Comment:
The usual practice is to delete commented out code before doing a PR.

Could we delete these lines?